cmake/partition_manager.cmake

```diff
@@ -11,22 +11,35 @@ if(FIRST_BOILERPLATE_EXECUTION)
     PARTITION_MANAGER_CONFIG_FILES
     )
 
-  if(partition_manager_config_files)
+  set(partitions_yml ${APPLICATION_SOURCE_DIR}/prj.partitions.yml)
```

Comment:
partitions_yml should be renamed to prj_partitions_yml or predefined_configuration_file

Reply:
static_configuration_file